src/ZipkinBundle/Components/Messenger/ZipkinHandlerInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+
+namespace ZipkinBundle\Components\Messenger;
+
+use Symfony\Component\Messenger\Envelope;
+use Zipkin\Tracing;
+
+interface ZipkinHandlerInterface
+{
+    public function __construct(Tracing $tracing);
```

Comment:
I am not sure about the constructor being part of the interface. I'd prefer implementation to be constructed in the way they want.